src/protocolsupport/zplatform/impl/spigot/entitytracker/SpigotEntityTrackerEntry.java

```diff
@@ -105,8 +105,8 @@
 	private boolean isPassenger;
 	private boolean onGround;
 
-	private final Set<EntityPlayer> trackedDefaultPlayers = new HashSet<>();
-	private final Set<EntityPlayer> trackedPEPlayers = new HashSet<>();
+	public final Set<EntityPlayer> trackedDefaultPlayers = new HashSet<>();
```

Comment:
Please make them protected. It's not the solution to allow access from outside of the class.